packages/airnode-admin/test/e2e/cli.feature.ts

```diff
@@ -211,6 +211,27 @@ describe('CLI', () => {
       expect(sponsored).toBe(true);
     });
 
+    it('uses transaction overrides', async () => {
```

Comment:
Could you also write a test that uses the admin SDK (for any command) and passes undefined as the third argument (because when used as a CLI, the `parseTransactionOverrides` will always return `{}` if none override is passed)

Reply:
I was thinking that here: [sponsorRequester](https://github.com/api3dao/airnode/pull/834/files#diff-df88b69abee08b2eb5c21f2807960839c363faa0260009f82eef94880ad6b217R231)  is called with omitted overrides, but I'll add a test for admin SDK as well.